src/summary/profile/age-helpers.tsx

```diff
@@ -0,0 +1,62 @@
+// Based on http://www.gregoryschmidt.ca/writing/patient-age-display-ehr-conventions,
+// which is different from npm packages such as https://www.npmjs.com/package/timeago
+export function age(dateString: string): string {
+  const today = new Date();
+  const birthDate = new Date(dateString);
+  const monthDifference = today.getUTCMonth() - birthDate.getUTCMonth();
+  const dateDifference = today.getUTCDate() - birthDate.getUTCDate();
+  let age = today.getUTCFullYear() - birthDate.getUTCFullYear();
+  if (monthDifference < 0 || (monthDifference === 0 && dateDifference < 0)) {
```

Comment:
There's a lot of weird little temporary variables in play here. If only JS supported memoization. Maybe see if there's an easy way to partition the code lines 4-23 into "intermediate values, for calculating display values" and "display values", with comments accordingly? Just an idea, not a big deal.

Reply:
I separated the code into sections and put comments in there. I'm not too worried about performance of this function right now -- happy to optimize with memoization / intermediate values if/when there's a measurable perf problem with it